catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfImageTransformer.java

```diff
@@ -152,21 +151,6 @@ protected void handleGeometry(ImageSegment imageSegmentHeader, List<Polygon> pol
   }
 
   /*
-   * Extracts the mission identifier from the image segment IID2 field
-   */
-  protected void handleMissionIdentifier(Metacard metacard, String imageIdentifier2) {
-    final int startIndex = 7; // inclusive
-    final int endIndex = 11; // exclusive
-
-    if (StringUtils.isNotBlank(imageIdentifier2) && imageIdentifier2.length() > endIndex) {
-      String missionId = imageIdentifier2.substring(startIndex, endIndex);
-
-      LOGGER.debug("Setting the metacard attribute [{}, {}]", Isr.MISSION_ID, missionId);
-      metacard.setAttribute(new AttributeImpl(Isr.MISSION_ID, missionId));
-    }
-  }
```

Comment:
This `Attribute` was moved to https://github.com/codice/alliance/blob/e7c036fa8a556d1175a55e69bcba6b64ca4ecbae/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageAttribute.java.